src/inputs_from_config.jl

```diff
@@ -60,13 +60,22 @@ function estimands_from_variants(
     estimand_constructor, 
     outcomes, 
     confounders;
+    extra_treatments=[],
     outcome_extra_covariates=[],
     positivity_constraint=0.,
     verbosity=1
     )
     estimands = []
     for variant in variants
-        treatments = treatments_from_variant(variant, dataset)
+       
+        if isempty(extra_treatments)
+            treatments = treatments_from_variant(variant, dataset)
+        elseif length(extra_treatments) == 1
+            treatments = Dict(treatments_from_variant(variant, dataset)..., treatments_from_variant(string(extra_treatments[1]), dataset)...)
+        else
+            error("GWIS mode only supports pairwise interaction with one extra treatment.")
```

Comment:
if someone tries to do a GWAS with extra treatment this error message will be misleading.